modules/rosa-sdpolicy-security.adoc

```diff
@@ -0,0 +1,48 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-service-definition.adoc
+
+[id="rosa-sdpolicy-security_{context}"]
+= Security
+
+== Authentication provider
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] or cluster creation process or using the `rosa` CLI. The `rosa` CLI allows the use of `htpasswd`. {product-title} is not an identity provider, and all access to the cluster must be managed by the customer as part of their integrated solution. Multiple identity providers provisioned at the same time is supported. The following identity providers are supported:
+
+- GitHub or GitHub Enterprise
+- GitLab
+- Google
+- LDAP
+- OpenID connect
+
+== Privileged containers
+Privileged containers are not supported on {product-title}. To enable Red Hat to operate {product-title} as a managed service with an SLA, some restrictions are enforced to limit the ability of rogue or accidental changes that could impact the service.
```

Comment:
Per ISG, use "might" or "can" instead of "could impact the service."

Reply:
Done.